communities/microgalaxy/lab/sections/3_microbiome.yml

```diff
@@ -1,9 +1,159 @@
+      description_md: >
+        An interactive browser application for analyzing and visualization metagenomics classification results from classifiers such as Kraken, KrakenUniq, Kraken 2, Centrifuge and MetaPhlAn. Pavian also provides an alignment viewer for validation of matches to a particular genome.
+      inputs:
+        - datatypes:
+            - Kraken report
+            - MetaPhlAn report
+      button_link: "{{ galaxy_base_url }}/root?tool_id=interactive_tool_pavian"
+    - title_md: <code>Phinch</code> - A framework for visualizing bio data
+      description_md: >
+        Interactive tool for visualising Biom data.
+      inputs:
+        - datatypes:
+            - Biom
+      button_link: "{{ galaxy_base_url }}/root?tool_id=interactive_tool_phinch"
+
+- id: databases
+  title: Databases
+  heading_md: Examples of databases prebuilt in some tools by microGalaxy
+  content:
+    - title_md: In <code>Kraken</code> tool
+      description_md: >
+        <ul>
+          <li> Viral </li>
+          <li> Plasmid </li>
+          <li> Archaea </li>
+          <li> Bacteria </li>
+        </ul>
+    - title_md: In <code>MetaPhlAn</code> tool
+      description_md: >
+        <ul>
+          <li> MetaPhlAn clade-specific marker genes (mpa_vOct22_CHOCOPhlAnSGB_202403) </li>
+          <li> MetaPhlAn clade-specific marker genes (mpa_vJun23_CHOCOPh|AnSGB_202403) </li>
+          <li> MetaPhiAn clade-specific marker genes (mpa_vOct22_CHOCOPhlAnSGB_202212) </li>
+          <li> MetaPhiAn clade-specific marker genes (mpa_vJan21_CHOCOPhlAnSGB_202103) </li>
+        </ul>
+    - title_md: In OTU analysis tools
+      description_md: >
+        <ul>
+          <li> Silva </li>
+          <li> Greengenes </li>
+          <li> RDP </li>
+        </ul>
+    - title_md: In <code>DADA2</code> tools
+      description_md: >
+        <ul>
+          <li> Silva </li>
+          <li> GTDB, Genome Taxonomy Database (Bacteria & Archaea) </li>
+          <li> Protist Ribosomal Reference database (PR2) </li>
+          <li> UNITE for Fungi </li>
+          <li> HItDB version 1 (Human InTestinal 16s rRNA) </li>
+          <li> NCBI RefSeq 16S rRNA database supplemented by RDP </li>
+        </ul>
+
 - id: workflows
   title: Workflows
-  heading_md: ''
+  heading_md: Examples of microbiome analysis workflows to use directly, modify or take ideas from
+  content:
+    - title_md: Metagenomic Taxonomy and Functional Analysis
+      description_md: Workflow covering operations related to Phylogenetic tree analysis,
+        Species frequency estimation, Visualisation, Aggregation, Phylogenetic analysis,
+        Read mapping, Standardisation and normalisation, Sequencing quality control,
+        Nucleic acid sequence analysis, Conversion, Sequence contamination filtering,
+        Taxonomic classification, Statistical calculation, Formattingon topics related
+        to nan      
+      button_link: https://usegalaxy.eu/published/workflow?id=cb2df493f5e334f7
```

Comment:
That will not work for a user on another server than usegalaxy.eu

Reply:
the link opens anyway, but to choose a workflow that is imported and all tools are there, I need to try to import it on all servers right?

Reply:
Have you tried?